File: moduleCore/pointCheck/src/main/java/com/einyun/app/pms/pointcheck/net/URLs.java
package com.einyun.app.pms.pointcheck.net;

/**
 * @ProjectName: pms_old
 * @Package: com.example.shimaostaff.pointcheck.net
 * @ClassName: URLs
 * @Description: java类作用描述
 * @Author: chumingjun
 * @CreateDate: 2019/10/08 15:12
 * @UpdateUser: 更新者：
 * @UpdateDate: 2019/10/08 15:12
 * @UpdateRemark: 更新说明：
 * @Version: 1.0
 */
public class URLs {
    //新增时获取该地块下的所有点检设置
    public static final String URL_POINT_CHECK_PROJECTS="resource/api/checkcenter/v1/checkRecord/getProject";

    //根据点检设置获取该点检设置下面的点检内容
    public static final String URL_POINT_CHECK_PROJECT_CONTENT="resource/api/checkcenter/v1/checkRecord/getContent/";

    //create new point check
    public static final String URL_POINT_CHECK_CREATE="resource/api/checkcenter/v1/checkRecord/add";

    //list of brief checkpoints
    public static final String URL_POINT_CHECK_LIST="resource/api/checkcenter/v1/checkRecord/list";

    public static final String URL_POINT_CHECK_DETIAL="resource/api/checkcenter/v1/checkRecord/get/";
}

File: modulePublic/einyunLibrary/src/main/java/com/einyun/app/library/dashboard/net/URLs.kt
package com.einyun.app.library.dashboard.net

/**
 *
 * @ProjectName:    android-framework
 * @Package:        com.einyun.app.library.dashboard.net
 * @ClassName:      URLs
 * @Description:     java类作用描述
 * @Author:         chumingjun
 * @CreateDate:     2019/11/4 0004 上午 11:19
 * @UpdateUser:     更新者
 * @UpdateDate:     2019/11/4 0004 上午 11:19
 * @UpdateRemark:   更新说明
 * @Version:        1.0
 */
class URLs {
    companion object {
        const val DOMAIN = "dashboard"
        //工单处理情况总览
        const val URL_DASHBOARD_WORK_ORDER_DATA = "/$DOMAIN/dashboard/api/workOrderData"
        //运营收缴率
        const val URL_DASHBOARD_OPERATE_CAPTURE_RATE = "/app/fee/proxy/api/fee/fee-center/api/getTotalRate"
        //首页权限配置
        const val URL_DASHBOARD_USER_MENU = "/portal/sys/sysMenu/v1/getCurrentUserMenu"
        //全部收费项目
        const val URL_ALL_CHARGED_PROJECT = "app/fee/proxy/api/fee/fee-center/api/getAmountDetail"
    }
}